tests/rptest/scale_tests/many_partitions_test.py

```diff
@@ -90,9 +90,11 @@ def __init__(self,
         if self.node_cpus >= 8:
             shard0_reserve = PARTITIONS_PER_SHARD / 2
 
-        # Reserve a few slots for internal partitions, do not be
-        # super specific about how many because we may add some in
-        # future for e.g. audit logging.
+        # Reserve a few slots for internal partitions. This is a cluster-wide
+        # value and doesn't include a replication factor (see per_node_slack).
```

Comment:
"doesn't include a replication factor" is confusing I think.

Maybe you can say "This is a count of partitions and we assume the replication factor is ... in order to calculate the associated number of partition replicas".

In general if we say "partitions" it is "pre rf multiplication" and if we say "partition replicas" it is after.